polyfills/WebAnimations/polyfill.js

```diff
@@ -0,0 +1,17 @@
+// Copyright 2014 Google Inc. All rights reserved.
```

Comment:
With automatically imported polyfills, we ignore them in git. Could you add this file to our .gitignore please?